estela_cli/init.py

```diff
@@ -85,22 +86,32 @@ def gen_dockerfile(requirements_path):
 
 @click.command(short_help=SHORT_HELP)
 @click.argument("pid", required=True)
+@click.option(
+    "-p",
+    "--platform",
+    default="scrapy",
+    help="Platform to use, it can be 'scrapy' or 'requests'",
+    show_default=True,
+)
 @click.option(
     "-r",
     "--requirements",
     default=DOCKER_DEFAULT_REQUIREMENTS,
     help="Relative path to requirements inside your project",
     show_default=True,
 )
-def estela_command(pid, requirements):
+def estela_command(pid, platform, requirements):
     """Initialize estela project
 
     PID is the project's pid
     """
-
+    platform_map = {
+        "scrapy": DOCKER_DEFAULT_ENTRYPOINT,
+        "requests": DOCKER_REQUESTS_ENTRYPOINT,
+    }
     if not os.path.exists(ESTELA_DIR):
         os.makedirs(ESTELA_DIR)
-
     estela_client = login()
-    gen_estela_yaml(estela_client, pid)
-    gen_dockerfile(requirements)
+    gen_estela_yaml(estela_client, platform_map[platform], pid)
+    gen_dockerfile(requirements, platform_map[platform])
+    click.echo(f"{pid} project is initialized as a {platform.capitalize()} project.")
```

Comment:
If I use `estela init <PID> -p whatever` the CLI breaks. Please specify allowed options for the `-p` option such as in https://github.com/bitmakerla/estela-cli/blob/09db075d5321e5186a7bb020d899fb8d05a605f0/estela_cli/data/job.py#L27